pkg/reconciler/volumeclaim/pvchandler.go

```diff
@@ -102,11 +102,11 @@ func getPersistentVolumeClaims(workspaceBindings []v1beta1.WorkspaceBinding, own
 // workspaceBinding name and ownerReference UID - because it is first used for creating a PVC and later,
 // possibly several TaskRuns to lookup the PVC to mount.
 // We use ownerReference UID over ownerReference name to distinguish runs with the same name.
-func GetPersistentVolumeClaimName(claim *corev1.PersistentVolumeClaim, wb v1beta1.WorkspaceBinding, owner metav1.OwnerReference) string {
-	if claim.Name == "" {
+func GetPersistentVolumeClaimName(claimName string, wb v1beta1.WorkspaceBinding, owner metav1.OwnerReference) string {
```

Comment:
can you update the docstring of this function? the docstring should also clarify the function's behavior when claimName is an empty string.

Reply:
String doc updated!